stable/aws-for-fluent-bit/templates/service.yaml

```diff
@@ -10,12 +8,10 @@ metadata:
 spec:
   ports:
   - name: monitor-agent
-    port: {{ .Values.serviceMonitor.service.port }}
+    port: {{ .Values.service.port }}
     protocol: TCP
-    targetPort: {{ .Values.serviceMonitor.service.targetPort }}
+    targetPort: {{ .Values.service.targetPort }}
   selector:
     {{- include "aws-for-fluent-bit.selectorLabels" . | nindent 4 }}
   sessionAffinity: None
-  type: {{ .Values.serviceMonitor.service.type }}
-{{- end }}
-{{- end }}
+  type: {{ .Values.service.type }}
```

Comment:
Sorry I do not understand this, why change from Values.serviceMonitor to the backwards breaking name Values.service??